types/auth0/index.d.ts

```diff
@@ -1157,11 +1157,11 @@ export interface GrantResponse {
 
 export class AuthenticationClient {
     // Members
-    database?: DatabaseAuthenticator | undefined;
-    oauth?: OAuthAuthenticator | undefined;
-    passwordless?: PasswordlessAuthenticator | undefined;
-    tokens?: TokensManager | undefined;
-    users?: UsersManager | undefined;
+    database: DatabaseAuthenticator;
+    oauth: OAuthAuthenticator;
+    passwordless: PasswordlessAuthenticator;
+    tokens: TokensManager;
+    users: UsersManager;
```

Comment:
so those also looks like read-only?, if not available to be set (we've just leaked details in types for no obvious gain for users)